flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptionsTest.java

```diff
@@ -29,7 +29,11 @@ public class RocksDBNativeMetricOptionsTest {
     public void testNativeMetricsConfigurable() {
         for (RocksDBProperty property : RocksDBProperty.values()) {
             Configuration config = new Configuration();
-            config.setBoolean(property.getConfigKey(), true);
+            if (property.getConfigKey().contains("num-files-at-level")) {
+                config.setBoolean("state.backend.rocksdb.metrics.num-files-at-level", true);
```

Comment:
```suggestion
                config.setBoolean(RocksDBNativeMetricOptions.MONITOR_NUM_FILES_AT_LEVEL.key(), true);
```

Reply:
@masteryhx Thank you for your suggestion, updated.